cli/Cargo.toml

```diff
@@ -30,3 +30,12 @@ serde_json = "1.0.55"
 structopt = { version = "0.3.15", default-features = false }
 
 reinfer-client = { version = "0.4.0", path = "../api" }
+
+[features]
+default = []
+
+# Enable experimental features that may break at any time.
+experimental = ["alpha"]
```

Comment:
The `experimental` feature is not used for anything, is it?

Reply:
I guess it's nice as a meta-feature so that you can just write `--features experimental` instead of `--features "alpha beta"` etc.